charts/fybrik/notebook-tests.values.yaml

```diff
@@ -30,6 +30,12 @@ opaServer:
         input.action.actionType == "write"
       }
 
+      rule[{}] {
+        description := "allow read datasets with no tags"
+        input.action.actionType == "read"
+        not input.resource.metadata.tags
```

Comment:
Does it work for an empty tags structure?

Reply:
Are you referring to the case where there is a tag structure but it is empty?  `manager/testdata/notebook/fybrikapplication-read.yaml` has no tags and it works fine for this notebook sample.